emcc.py

```diff
@@ -1444,10 +1454,11 @@ def get_final():
     with ToolchainProfiler.profile_block('post-link'):
       if DEBUG:
         logging.debug('saving intermediate processing steps to %s', shared.get_emscripten_temp_dir())
-        if not LEAVE_INPUTS_RAW: save_intermediate('basebc', 'bc')
+        if not LEAVE_INPUTS_RAW and not shared.Settings.WASM_BACKEND:
+          save_intermediate('basebc', 'bc')
 
       # Optimize, if asked to
-      if not LEAVE_INPUTS_RAW:
+      if not LEAVE_INPUTS_RAW and not shared.Settings.WASM_BACKEND:
```

Comment:
ditto, looks like this would disable lto for the wasm backend (when not using wasm object files)